packages/core/src/controllers/subscriber.ts

```diff
@@ -37,7 +39,7 @@ export class Subscriber extends ISubscriber {
 
   private cached: SubscriberTypes.Active[] = [];
   private initialized = false;
-
+  private pendingSubscriptionWatchLabel = PENDING_SUB_WATCH_LABEL;
```

Comment:
Do we need to define an shared external constant here if the label value is only used in the subscriber and is private?